learning_journal/views/default.py

```diff
@@ -40,7 +52,7 @@ def create_view(request):
         return {"creation_date": "{}-{}-{}".format(today.year, today.month, today.day)}
```

Comment:
Why have an if block for request.method==GET that returns a creation date.  You assign a value to entry object's creation data attribute on a POST and never use this so why have it?